source-connector/src/test/kotlin/org/neo4j/connectors/kafka/source/Neo4jCdcSourceValueStrategyIT.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [https://neo4j.com]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.connectors.kafka.source
+
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import java.time.Duration
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInfo
+import org.neo4j.cdc.client.model.ChangeEvent
+import org.neo4j.cdc.client.model.EntityOperation
+import org.neo4j.cdc.client.model.EventType
+import org.neo4j.connectors.kafka.testing.MapSupport.excludingKeys
+import org.neo4j.connectors.kafka.testing.assertions.ChangeEventAssert
+import org.neo4j.connectors.kafka.testing.assertions.TopicVerifier
+import org.neo4j.connectors.kafka.testing.format.KafkaConverter.AVRO
+import org.neo4j.connectors.kafka.testing.format.KafkaConverter.JSON_EMBEDDED
+import org.neo4j.connectors.kafka.testing.format.KafkaConverter.JSON_SCHEMA
+import org.neo4j.connectors.kafka.testing.format.KafkaConverter.PROTOBUF
+import org.neo4j.connectors.kafka.testing.format.KeyValueConverter
+import org.neo4j.connectors.kafka.testing.kafka.ConvertingKafkaConsumer
+import org.neo4j.connectors.kafka.testing.source.CdcSource
+import org.neo4j.connectors.kafka.testing.source.CdcSourceParam
+import org.neo4j.connectors.kafka.testing.source.CdcSourceTopic
+import org.neo4j.connectors.kafka.testing.source.Neo4jSource
+import org.neo4j.connectors.kafka.testing.source.SourceStrategy
+import org.neo4j.connectors.kafka.testing.source.TopicConsumer
+import org.neo4j.driver.Session
+
+abstract class Neo4jCdcSourceValueStrategyIT {
+  @Neo4jSource(
+      startFrom = "EARLIEST",
+      strategy = SourceStrategy.CDC,
+      cdc =
+          CdcSource(
+              topics =
+                  arrayOf(
+                      CdcSourceTopic(
+                          topic = "neo4j-cdc-topic-value-serialization-change-event",
+                          patterns = arrayOf(CdcSourceParam("(:TestSource{name,+execId})")),
+                          valueSerialization = "CHANGE_EVENT"),
+                  ),
+          ),
+  )
+  @Test
+  fun `supports serialization of values as whole change event`(
+      testInfo: TestInfo,
+      @TopicConsumer(
+          topic = "neo4j-cdc-topic-value-serialization-change-event", offset = "earliest")
+      consumer: ConvertingKafkaConsumer,
+      session: Session
+  ) {
+    val executionId = testInfo.displayName + System.currentTimeMillis()
```

Comment:
do we still need this in tests?

Reply:
Nope, tests still run successfully. Removing it from both key and value strategy tests.